index.html

```diff
@@ -12,9 +12,11 @@
     />
   </head>
   <body>
-    <h1>Awesome Books</h1>
+    <h1 class="text-center">All Awesome Books</h1>
```

Comment:
- [Optional ] Please change the font color so the user can see the book title clearly 

![image](https://user-images.githubusercontent.com/74744787/164078813-9f173c4e-0b25-4956-b85a-08ae865c19f9.png)

Reply:
Everything is fine on my and my partner's side, please check your browser maybe you're using an extension that enables the dark mode.